### ISSUE TITLE
Support launch storyboards as splash screens on iOS

### ISSUE THREAD

**Issue by abarisic86** [crazycodeboy/react-native-splash-screen#115]
Is launch storyboard supported with this library? I can't get it to work.

**Comment by maggialejandro** [comment]
I'm having the same problem trying to configure iPhone X following this [tutorial](https://blog.callstack.io/react-native-meets-iphonex-a0f5034f7a41)
The splashscreen stoped working after I added the storyboard..

**Comment by jordanmkoncz** [comment]
Also wondering this, as my app is using a LaunchScreen.storyboard file.

**Comment by wincod75** [comment]
Anyone still having this issue? Doesn't seem to play nice with .storyboard files; won't unmount the splash screen, just stays on top.

**Comment by gwmccull** [comment]
anyone know if this is an issue? Apparently in April 2020, we'll all be required to use Storyboard
https://developer.apple.com/news/?id=01132020b

**Comment by gwmccull** [comment]
update, I switched us to using a storyboard w/o issues. The only change was to create the storyboard and then change the project target to use that as the launch screen instead of the old `LaunchScreen.xib`

**Comment by Victorien-Tardif** [comment]
> update, I switched us to using a storyboard w/o issues. The only change was to create the storyboard and then change the project target to use that as the launch screen instead of the old `LaunchScreen.xib`

Hello @gwmccull, how did you do that ? I'm confused, I just can't see the reference to LaunchScreen.xib, now that I updated to a .storyboard file I can't make it work, splashscreen is always white

**Comment by gwmccull** [comment]
@Victorien-Tardif did you change the Launch Screen File setting on your target?
![image](https://user-images.githubusercontent.com/580060/75834681-91fa2980-5d71-11ea-9b49-96abee974084.png)

**Comment by Victorien-Tardif** [comment]
> @Victorien-Tardif did you change the Launch Screen File setting on your target?
> ![image](https://user-images.githubusercontent.com/580060/75834681-91fa2980-5d71-11ea-9b49-96abee974084.png)
> 
> select your project, then select the correct target, then choose the General tab

Yes of course ! But it does not work at all. No changes to make for RNSplashScreen ?

**Comment by Victorien-Tardif** [comment]
> > @Victorien-Tardif did you change the Launch Screen File setting on your target?
> > ![image](https://user-images.githubusercontent.com/580060/75834681-91fa2980-5d71-11ea-9b49-96abee974084.png)
> > select your project, then select the correct target, then choose the General tab
> 
> Yes of course ! But it does not work at all. No changes to make for RNSplashScreen ?

Well, for some reason it works when I put the file in the Base.lproj folder!

**Comment by MoOx** [comment]
In a month that is going to be a real issue.

> Starting April 30, 2020, apps submitted to the App Store must use an Xcode storyboard to provide the app’s launch screen

Source: https://developer.apple.com/ios/submit/

To be ready, you should start using https://github.com/zoontek/react-native-bootsplash that already support storyboard

**Comment by timhatch** [comment]
I found the tutorial at [React Native iOS Splash Screen with Storyboard](https://blog.echobind.com/react-native-ios-splash-screen-with-storyboard-f6f9d847994e) quite helpful.
I may look at the [BootSplash](https://github.com/zoontek/react-native-bootsplash) component for a new build.

**Comment by bohomi** [comment]
replacing the RNSplashScreen line in `AppDelegate.m` did the trick for me.

changed from `[RNSplashScreen showSplash:@"LaunchScreen" inRootView:rootView];` to `[RNSplashScreen show];`

**Comment by YOEL311** [comment]
@bohomi 

I would like to know how the show method works.
Which file should be loaded?

There is code for the library, but I don't understand how it works
``[RNSplashScreen show];``

https://github.com/crazycodeboy/react-native-splash-screen/blob/b47197626804a742b8569cad50d5e0ed92fc765c/ios/RNSplashScreen.m#L23
